packages/inference/src/providers/fal-ai.ts

```diff
@@ -0,0 +1,17 @@
+import type { ModelId } from "../types";
+
+export const FAL_AI_API_BASE_URL = "https://fal.run";
+
+type FalAiId = string;
+
+/**
+ * Mapping from HF model ID -> fal.ai app id
+ */
+export const FAL_AI_MODEL_IDS: Record<ModelId, FalAiId> = {
+	/** text-to-image */
+	"black-forest-labs/FLUX.1-schnell": "fal-ai/flux/schnell",
+	"black-forest-labs/FLUX.1-dev": "fal-ai/flux/dev",
+
+	/** automatic-speech-recognition */
+	"openai/whisper-large-v3": "fal-ai/whisper",
+};
```

Comment:
Idk if it has been discussed somewhere but I feel that at some point these mappings should be maintained as an API lazy-loaded by the inference client (same as what we are doing for the tasks). 
This way, we would be able to retrieve it in `huggingface_hub` + could be quickly updated without client releases.

Reply:
scroll up: https://github.com/huggingface/huggingface.js/pull/1077#discussion_r1908758678 :)

Reply:
🙈

Reply:
great minds think alike